posthog/models/cohort.py

```diff
@@ -283,7 +283,13 @@ def get_and_update_pending_version(cohort: Cohort):
 class CohortPeople(models.Model):
     id: models.BigAutoField = models.BigAutoField(primary_key=True)
     cohort: models.ForeignKey = models.ForeignKey("Cohort", on_delete=models.CASCADE)
-    person: models.ForeignKey = models.ForeignKey("Person", on_delete=models.CASCADE)
+    person: models.ForeignKey = models.ForeignKey(
```

Comment:
Is there any possibility that we will access a related person object that doesn’t exist? In that case Django will raise a `DoesNotExist` exception.

Reply:
Good question, I was expecting persons to be tombstoned rather than DELETE'd but I'll check if that's actually the case.